calc/calc.py

```diff
@@ -52,6 +58,72 @@ def __init__(self, text, position=0, current_token=None):
         self.position = position
         self.current_token = current_token
 
+    def _tokenize_integer(self, text):
+        """If have found a single digit then this helper function will read
+        characters until the end of the integer is found.
+
+        args:
+            text: A section of the text that is assumed to contain an integer
+
+        Returns:
+            Token
+
+        Raises:
+            None
+        """
+        # Saved By The Test: The first couple attempts at this function were
+        # very messed up  and screwed up other functionality in the interpreter
+        # that my tests  caught.
+
+        # First we check to see if there are other digits in the int.
+        first_position = self.position
+        current_position = self.position
+        current_character = text[current_position]
+
+        while True:
+            if not current_character.isdigit():
+                break
+
+            current_position += 1
+            try:
+                current_character = text[current_position]
+            except IndexError:
+                # If we've hit EOF we'll hit this exception.
+                break
+
+        # Current position should be EOF or a digit.
+        self.position = current_position
+        value = int(text[first_position: current_position])
+        return Token(INTEGER, value)
+
+    def _consume_whitespace(self, text):
+        """Eats all whitespace found until there's nothing left.
+
+        args:
```

Comment:
Consistency in capitalization across all docstrings. I don't particularly care if its capitalized or not as much as I care about the consistency.